src/pages/sprikipedia/getting-started/about-sprig.astro

```diff
@@ -31,4 +31,4 @@ const data = (
       name="CodeMirror"
       description=" " />
 </div>
-</Categories>
```

Comment:
Disagree with the rename, getting-started folder follow README.md, and in README.md is https://github.com/hackclub/sprig#acknowledgements